hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -43,6 +44,40 @@
 from hpat.utils import to_array
 
 
+class TypingChecker:
+    msg_template = '{} The object must be {}. Given{}: {}'
```

Comment:
```suggestion
    msg_template = '{__func_name__} The object {__param__}\n  given: {__given_type__}\n  expected: {__expected_type__}'
```
or something like this to make it more clear.